lib/linguist/languages.yml

```diff
@@ -2986,6 +2988,12 @@ Myghty:
   tm_scope: none
   ace_mode: text
   language_id: 239
+Nand2Tetris Hack:
+  type: data
+  ace_mode: text
```

Comment:
You'll need to add `tm_scope: none` to tell Linguist we don't have a syntax highlighting grammar for that language.

Reply:
:+1: